### ISSUE TITLE
systemd-run: --property=WantedBy is added to the [Unit] section

### ISSUE THREAD

**Issue by daandemeyer** [systemd/systemd#14251]
**systemd version the issue has been seen with**
> v244

**Used distribution**
 > Ubuntu Focal

**Expected behaviour you didn't see**
 > WantedBy property was added to the [Install] section in the unit file.

**Unexpected behaviour you saw**
 > WantedBy property was added to the [Unit] section in the unit file.

**Steps to reproduce the problem**

```
systemd-run --pty --unit=cat --property=WantedBy=multi-user.target cat
cat /run/systemd/transient/cat.service
```

I'm trying to get systemd-run to generate unit files that I can copy to a persistent location and enable until (hopefully) a `--persist` option is added to systemd-run which is how I discovered the issue.

**Comment by keszybz** [comment]
Sorry, `systemd-run` is not a mechanism to template unit files. If you need that, use jinja2 or something like that.

**Comment by DaanDeMeyer** [comment]
@keszybz Shouldn't systemd-run block properties like `WantedBy` then?

**Comment by keszybz** [comment]
Indeed.

**Comment by boucman** [comment]
I'd argue that since systemd-run can create timers/paths/sockets, having it create transient wanted-by/ would be coherent with how it works.

Still not an [Install] section, but another way to create a service, not start it immediately but plug-it into the existing system...

**Comment by Eucyon** [comment]
I am also looking for a way to create a simple service in an cli one-liner, as a kind of at replacement.

Is it possible? If not, wouldn't it make sense?

**Comment by poettering** [comment]
Fix waiting in #14528.
